src/lib/MUIRating.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { IconContainerProps, Rating, RatingProps } from '@material-ui/lab'
+import { Box, Typography } from '@material-ui/core';
+import { attachField, IFieldProps } from 'react-forms'
+import { FormikValues } from 'formik';
+import _ from 'lodash';
+
+export interface IMUIRating extends RatingProps {
+	icons?: JSX.Element[]
+	labels?: string[]
+	description?: string
+}
+
+export interface IProps extends IFieldProps {
```

Comment:
Change IProps to RatingProps, to avoid conflict with Material UI Rating prop, change it's import statement as follows.

import {RatingProps as MaterialRatingProps} from '@material-ui/labs';